src/tooltip/tooltip.js

```diff
@@ -319,6 +321,12 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
               ttScope.popupDelay = ! isNaN(delay) ? delay : options.popupDelay;
             }
 
+            function prepPopupCloseDelay() {
+              var val = attrs[ prefix + 'PopupCloseDelay' ];
+              var delay = parseInt( val, 10 );
```

Comment:
@wesleycho I'm staying consistent with the rest of the file, if you want me to update the whole file then I can.